packages/core/src/pages/[...slug].tsx

```diff
@@ -91,12 +93,29 @@ export const getStaticProps: GetStaticProps<
   ]
 
   if (await landingPagePromise) {
+    // Checking if the fetch function corresponding to the slug exists
+    const fetchFunction = dynamicContent[slug]
+    let serverData = null
+
+    if (!fetchFunction) {
+      console.warn(`Warning: Fetch function not found for slug: ${slug}`)
```

Comment:
question: Can we give any orientation regarding where/what to check? (missing function/error when mapping)

Reply:
I've added more instructions [here](https://github.com/vtex/faststore/pull/2242/commits/7bdb9a39a780a5bbc07e23a513ac2378531817e8). What do you think?

Reply:
Niceeee! 💯  I like when we can give further help.
Congrats on the doc!